### PR TITLE
DOC-33: MyAvatar API JSDoc catch-up

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Update JSDoc per recently added MyAvatar and Avatar properties and methods.

Case: https://highfidelity.atlassian.net/browse/DOC-33

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19767/HighFidelity-Beta-PR15676-f91c6de.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19767/HighFidelity-Beta-Interface-PR15676-f91c6de.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19767/HighFidelity-Beta-PR15676-f91c6de.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19767/HighFidelity-Quest-Beta-PR15676-f91c6de.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19767/HighFidelity-Beta-PR15676-f91c6de.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19767/HighFidelity-Beta-Interface-PR15676-f91c6de.exe

File: interface/src/avatar/MyAvatar.h [hunk 1]
```diff
@@ -39,6 +39,23 @@ class ModelItemID;
...
+ * <table>
+ *   <thead>
+ *     <tr><th>Value</th><th>Name</th><th>Description</th></tr>
+ *   </thead>
+ *   <tbody>
+ *     <tr><td><code>0</code></td><td>Default</td><td>Your walking speed is constant; it doesn't change depending on how far 
+ *       forward you push your controller's joystick. Fully pushing your joystick forward makes your avatar run.</td></tr>
+ *     <tr><td><code>1</code></td><td>Analog</td><td>Your walking speed changes in steps based on how far forward you push your 
+ *       controller's joystick. Fully pushing your joystick forward makes your avatar run.</td></tr>
+ *     <tr><td><code>2</code></td><td>AnalogPlus</td><td>Your walking speed changes proportionally to how far forward you push
```
Comment (jennaingersoll) [inline interface/src/avatar/MyAvatar.h]: This is dependent on the analogPlusWalkSpeed property.

File: interface/src/avatar/MyAvatar.h [hunk 2]
```diff
@@ -639,16 +661,15 @@ class MyAvatar : public Avatar {
     Q_INVOKABLE void restoreAnimation();
 
     /**jsdoc
-     * Restores the default hand animation state machine that is driven by the state machine in the avatar-animation json.
+     * Restores the default hand animation state machine that is driven by the state machine in the avatar-animation JSON.
      * <p>The avatar animation system includes a set of default animations along with rules for how those animations are blended
      * together with procedural data (such as look at vectors, hand sensors etc.). Playing your own custom animations will
      * override the  default animations. <code>restoreHandAnimation()</code> is used to restore the default hand poses
```
Comment (jennaingersoll) [inline interface/src/avatar/MyAvatar.h]:
```suggestion
     * override the  default animations. <code>restoreHandAnimation()</code> is used to restore the default hand poses.
```

Comment (ctrlaltdavid) [reply interface/src/avatar/MyAvatar.h]: Fixed.

File: interface/src/avatar/MyAvatar.h (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/avatar/MyAvatar.h]: No change.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19792/HighFidelity-Beta-PR15676-caf29dc.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19792/HighFidelity-Quest-Beta-PR15676-caf29dc.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19792/HighFidelity-Beta-PR15676-caf29dc.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19792/HighFidelity-Beta-Interface-PR15676-caf29dc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19792/HighFidelity-Beta-PR15676-caf29dc.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19792/HighFidelity-Beta-Interface-PR15676-caf29dc.exe

**Pull request merged**